### ISSUE TITLE
nice meme

### ISSUE THREAD

**Issue by NickdogeDev** [clydetm/most-useless-bot-to-ever-exist#1]
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Comment by clydetm** [comment]
I really like pizza. It's a problem. It started off innocently enough. I was introduced to this amazing food as a child. Pizza would happen once in a while but it was beyond my control, so I didn't think much of it. However, I was already displaying warning signs. I would grab way more than my share of the pizza. Eventually I started getting in trouble for it, so I would drop it on the ground first and then eat it. Everyone thought it was gross and wouldn't eat it but I still loved it so I got extra pizza. Eventually my parents stopped ordering pizza after this happened a few times.
Later on, when I went to college, I was suddenly struck with the realization that I could have pizza *whenever I wanted*. It started off innocently enough; I'd order a pizza or two and split the costs with friends and we'd all eat some once a week. This wasn't enough for me, and I started ordering a personal pizza once per day. I would try and hide it in my bookbag or whatever I had handy because I didn't want my roommates to see. Once I got started, it was hard to stop. I ordered up to three large pizzas every day. My money ran out, and I would literally beg friends for money, letting them borrow my car or whatever I had to do, and then go spend it on pizza. The next semester, I took out huge student loans, and spent it all on pizza. I would start the semester with a few thousand dollars but a few months in and I had nothing. The local Dominos knew me by name. I can safely say that the majority of my student loan debt probably came from pizza.
Despite my pizza obsession and financial problems, I graduated university and started working, but my love for pizza had not changed. My first job out of college was making $60k a year. I'd spend money on the necessities, and the rest on pizza. There was a delicious local pizza place that had the best tasting stuff. I worked out a deal where they would deliver a pizza to me every day about an hour after work. A few months of this, though, and I needed to expand. I had tried every type of pizza out there, and I just wanted something better. I started making my own pizzas. I would go to the store and buy a few pounds of pizza dough, and on Sunday I would prepare as many pizzas as I could fit in my fridge, and make them throughout the week. I later referred to this as my 'artistic' pizza phase. I would try everything, from gouda cheese with ranch sauce to literally putting several hamburgers on top of my pizza. I tried different cooking methods, sometimes burning the crust to a crisp and leaving a sort of 'raw middle' kind of like sushi.
Even when I wasn't making pizza, I was thinking about it. I had pizza posters on my walls, even my desktop wallpaper was a delicious looking slice of pizza. I ended up spending so much on pizza that I figured it would actually be cheaper to work for a pizza place. I got a job at my local Dominos during the nights to supplement my pizza. I would have homemade pizza for breakfast and lunch and then I'd sneak some Domino's while I worked for dinner. I actually got fired from Domino's pretty fast because I was taking slices from other people's pizzas before I put them on the shelf, and they found me out.
At this time I was doing a lot of thinking with my life and what I wanted to do, and I decided since I loved pizza so much, I should open up a pizza shop! This is probably the worst decision I ever made. I opened up huge business loans, invested in a building and ingredients. I decided to spend about a month before hand making sure I knew how to use the equipment before I opened. It was amazing! A giant, stone fired pizza oven, fresh ingredients from my produce supplier, etc. I ended up eating all of my initial ingredients, and quickly ordered more. On the day I was supposed to open, I couldn't do it, and I kept the doors locked and decided to just use this place as my personal pizzeria. I broke my apartment lease, brought my stuff over, and lived in my pizzeria. Every day I was surrounded by the smell of pizza and all the fresh ingredients, it was probably the happiest I have ever been in my life.
However, the place didn't have showers or a bathroom, and I eventually became a huge mess. I smelled like B.O. and cheese, and my hygiene suffered to the point where I was fired from my good job. With the credit card debt looming, I choose to ignore my problems and bury myself in as much pizza as I could. Eventually a foreclosure guy came to get rid of me from my store, and he ended up calling the cops after I threw slices of pizza at him, telling him to leave me alone.
Anyway, that's about it. I'm homeless and in large amounts of debt. I have to panhandle on the sides of the street. I keep trying to save up some money to get my life back on track but I end up blowing it on a big cheesy pizza. The local Domino's now calls me the homeless guy, and I'm a regular there. I can't really say I regret my decisions, but sometimes I wish I could be a normal guy.
TL;DR: I like Pizza.

**Comment by NickdogeDev** [comment]
Yes, i tried turning it on and off again.

**Comment by clydetm** [comment]
Get Kyle's Chicken by going to the official admemes forums website.

After you have Kyle's Chicken, you must obtain Clyde's salt and vinegar chips.
NOTE: This is not easy to do! You must tell Clyde to stop maining Mercy on Overwatch, and main a better hero, like Hanzo!

When you have obtained both the Chicken and the chips, you must mash chip bits and raw chicken together and stuff it in Jake's oven for 20 minutes.

After 20 minutes, the bot should should smell proper. This is a good sign, as your versatile bot is ready!

**Comment by NickdogeDev** [comment]
ok

**Comment by NickdogeDev** [comment]
messed up again help plz

**Comment by NickdogeDev** [comment]
Helo?

**Comment by NickdogeDev** [comment]
Helo??

**Comment by NickdogeDev** [comment]
You want see?

**Comment by NickdogeDev** [comment]
Helo??????

**Comment by clydetm** [comment]
# 

> [`_****_`](

- 

- [ ] @ #1. url

)
